projects/demo/src/pages/kit/number/number-mask-doc.template.html

```diff
@@ -118,6 +118,28 @@
             </ng-template>
             <number-mask-doc-example-5></number-mask-doc-example-5>
         </tui-doc-example>
+
+        <tui-doc-example
+            id="custom-minus-sign"
+            heading="Custom minus sign"
+            [content]="customMinusSignExample6"
+            [description]="customMinusSignDescription"
+        >
+            <ng-template #customMinusSignDescription>
+                <p>
+                    You can use
+                    <code>minusSign</code>
+                    option for your convenience in working with negative numbers.
+                </p>
+                <p>
+                    In this exapmle
+                    <code>Katakana-Hiragana Prolonged Sound Mark</code>
```

Comment:
Let's use hyphen for this example.

It will be the most popular case

Reply:
Changed